### ISSUE TITLE
Tour save/undo actions in the toolbar are not accessible when "Tour Editor" view is detached in a separate window

### ISSUE THREAD

**Issue by wolfgang-ch** [wolfgang-ch/mytourbook#371]
These actions disappear when the main window has the focus.

Workaround: Ctrl+S and Ctrl+Z are still working
